### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by senavarro**
Updated everything to change 'fp' to 'lfa'

### CODE REVIEW

**Review by jjagandevaraj — DISMISSED** [review]
The CLI might be the significant fall-out of this, we need to keep an eye out.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### Study Notes
 
-[git CLI cheatsheet](https://github.com/getfutureproof/fp_guides_wiki/wiki/git-CLI-Cheatsheet)
+[git CLI cheatsheet](https://github.com/getfutureproof/lfa_guides_wiki/wiki/git-CLI-Cheatsheet)
```
Comment (rom-30) [inline README.md:3]: Can you please change this link back to `[git CLI cheatsheet](https://github.com/getfutureproof/fp_guides_wiki/wiki/git-CLI-Cheatsheet)`

File: README.md [hunk 2]
```diff
@@ -14,22 +14,22 @@ _Head on down to [Returners](#returners) for tips on how to make this smooth for
 ### 1. Fork & Clone
 
 - Hit 'Fork' on this repo (it's in the top right!)
-- Make sure you are now looking at **your fork** (it will say `<your-github-username>/fp_study_notes_hello_github` at the top of the page)
+- Make sure you are now looking at **your fork** (it will say `<your-github-username>/lfa_study_notes_hello_github` at the top of the page)
 - Click the green 'Code' button and copy the SSH option if you have already setup git in your terminal, or the HTTPS option if not.
 - Clone down **your fork** using `git clone <what you copied>` in your terminal
 - Move into the project folder with `cd fp_study_notes_hello_github`
```
Comment (rom-30) [inline README.md:20]: Can you please change this line to `cd lfa_study_notes_hello_github`

**Review by rom-30 — CHANGES_REQUESTED** [review]
Thank you Sergi :)

File: README.md (hunk 1 above)
Comment (rom-30) [reply README.md:3]: Sorry to be a pain, but the resource does not exist - We have not updated the wiki yet, so until we do, the only way for them to have access the cheatsheet is via https://github.com/getfutureproof/fp_guides_wiki/wiki/git-CLI-Cheatsheet


image

**Review by rom-30 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (senavarro) [reply README.md:3]: True... the wiki is for the old curriculum though, so we will need to think about how to proceed on this. @TDG-Lafosse what do you think?

**Comment by rom-30** [comment]
An updated README has been provided when the repo was transferred @senavarro :)

**Pull request closed without merging**